stdlib/2and3/fractions.pyi

```diff
@@ -39,33 +39,33 @@ class Fraction(Rational):
     def numerator(self) -> int: ...
     @property
     def denominator(self) -> int: ...
-    def __add__(self, other): ...
-    def __radd__(self, other): ...
-    def __sub__(self, other): ...
-    def __rsub__(self, other): ...
-    def __mul__(self, other): ...
-    def __rmul__(self, other): ...
-    def __truediv__(self, other): ...
-    def __rtruediv__(self, other): ...
+    def __add__(self, other: _ComparableNum) -> Fraction: ...
```

Comment:
This isn't true:
```
In [49]: Fraction(1, 2) + pi
Out[49]: 3.641592653589793
```
And `Fraction(1, 2) + Decimal(5)` throws; it looks like it just supports int, float, Fraction, and complex as the argument type.

Maybe we can do this correctly with overloads, but it's also fine to just leave the return types for most of these as Any.